test/Mvc.Api.Analyzers.Test/AddResponseTypeAttributeCodeFixProviderIntegrationTest.cs

```diff
@@ -33,6 +33,18 @@ public class AddResponseTypeAttributeCodeFixProviderIntegrationTest
         [Fact]
         public Task CodeFixAddsFullyQualifiedProducesResponseType() => RunTest();
 
+        [Fact]
+        public Task CodeFixAddsNumericLiteralForNonExistingStatusCodeConstants() => RunTest();
```

Comment:
This test case should ideally have been submitted with #8234, but because returning `StatusCodeResult` wasn't supported, I didn't see a good way to write it, which lead me down the path of writing this pull request 😝